harness/features/variable.local.test.ts

```diff
@@ -550,6 +550,7 @@ describe('Variable Tests - Local', () => {
                             etag: null,
                             rayId: null,
                             lastModified: null,
+                            skipSDKConfigEvent: true,
```

Comment:
just to confirm, this is because of the client not being initialized?

Reply:
yea